include/lcp-widget.php

```diff
@@ -58,6 +60,7 @@ function widget($args, $instance) {
       'excerpt_size' => $excerptsize,
       'exclude' => $exclude,
       'excludeposts' => $excludeposts,
+      'includeposts' => $includeposts,
```

Comment:
This is unnecessary. Actually this whole array here is kind of obsolete because of the merge I added on line 73